pom.xml

```diff
@@ -129,21 +129,34 @@
 				</executions>
 			</plugin>
 			<plugin>
-				<groupId>org.sonatype.plugins</groupId>
-				<artifactId>nexus-staging-maven-plugin</artifactId>
-				<version>1.6.3</version>
+				<groupId>org.apache.felix</groupId>
```

Comment:
Adding in the maven-bundle-plugin enables the `bundle` packaging type. It automatically generates an OSGi manifest (some additional properties stored in `/META-INF/MANIFEST.FM`).

The artifact itself is still a JAR and is otherwise unchanged.